tests/torch/fx/test_model_transformer.py

```diff
@@ -136,6 +152,78 @@ def test_output_insertion_transformation(tuple_output, target_point):
     )
 
 
+def torchvision_model_case(model_id: str, input_shape: Tuple[int,]):
+    model = getattr(models, model_id)
+    return ModelCase(partial(model, weights=None), model_id, input_shape)
+
+
+TEST_MODELS_QUANIZED = (
+    (torchvision_model_case("resnet18", (1, 3, 224, 224)), {}),
+    (torchvision_model_case("mobilenet_v3_small", (1, 3, 224, 224)), {}),
+    (torchvision_model_case("vit_b_16", (1, 3, 224, 224)), {"model_type": nncf.ModelType.TRANSFORMER}),
+    (torchvision_model_case("swin_v2_s", (1, 3, 224, 224)), {"model_type": nncf.ModelType.TRANSFORMER}),
+)
+
+
+@pytest.mark.parametrize(
+    ("model_case", "quantization_parameters"), TEST_MODELS_QUANIZED, ids=[m[0].model_id for m in TEST_MODELS_QUANIZED]
+)
+def test_post_quantization_compression(model_case: ModelCase, quantization_parameters):
+    with disable_patching():
+        torch.manual_seed(42)
+        model = getattr(models, model_case.model_id)().eval()
+        input_ids = torch.ones(model_case.input_shape)
+        exported_model = capture_pre_autograd_graph(model, args=(input_ids,))
+        quantization_parameters["advanced_parameters"] = nncf.AdvancedQuantizationParameters()
+        quantization_parameters["subset_size"] = 1
+        quantized_model = nncf.quantize(
+            exported_model, calibration_dataset=nncf.Dataset([input_ids]), **quantization_parameters
+        )
+
+    for node in quantized_model.graph.nodes:
+        if node.name[:3] == "mul":
+            input_tup = []
+            input_tup = _get_node_inputs(node, quantized_model)
+            if input_tup:
```

Comment:
What if all `input_tup` is None? Test will be green anyway. Please check amount of nodes here

Reply:
Done!